### PR TITLE
iroh: Add version 0.21.0

### PR DESCRIPTION
**Pull request by xialvjun**
Another tool created by IPFS team

Closes #XXXX

Relates to #XXXX

- [ ] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by pratikpc** [comment]
@xialvjun the 0.13.0 has not been marked as latest by the iroh team

That's why Checkver failed.

You will have to switch to 0.12 probably.

**Comment by pratikpc** [comment]
Should probably work now

/verify

File: bucket/iroh.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/n0-computer/iroh/releases/download/v0.13.0/iroh-windows-amd64.zip",
+            "hash": "08ded7cd4452cfeb8861134b6b91e356d687b6935f3df9b1f91e70fa9e932d1d"
+        }
+    },
+    "bin": "iroh.exe",
+    "checkver": {
+        "github": "https://github.com/n0-computer/iroh"
+    },
```
Comment (aliesbelik) [inline bucket/iroh.json:15]:
```suggestion
    "checkver": "github",
```

**Review by aliesbelik — COMMENTED** [review]
Will work since GH url already used as homepage.

**Pull request merged**